### DISCUSSION TITLE
Why is spotube removed from google play?

### DISCUSSION THREAD

**Discussion by ServerDeveloper9447** [KRTirtho/spotube#2160] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/eebec79d-cd37-424a-84fc-5685e043d927)

**Comment by KRTirtho** [comment, marked as answer]
It's temporarily suspened for "Repetitive Content Policy" violation. So it's an In app experience policy where content shouldn't repeated among multiple apps from the same dev, or in the same app in multiple places

I guess I've to make some the pages more distinguishable from each other

**Reply by KRTirtho** [reply]
Now it's permanently suspended. That's Google Play for y'all. 
I'll recommend to directly download from our official [website](https://spotube.krtirtho.dev/) or from F-Droid
> Please avoid apkpure, uptodown, apkmod etc as they're uploaded by 3rd party. It can be modified to include malware
